test/test_environment.py

```diff
@@ -586,7 +586,7 @@ def test_pypy_virtualenv(tmpdir):
     for env in environments:
         env.create()
         output = env.run(['-c', 'import sys; print(sys.pypy_version_info)'])
-        assert output.startswith(str("(major="))
+        assert "(major=" in output
```

Comment:
There was this PR for this error: #1002 I was waiting to have pypy tests in the CI to merge, happy to get merge there now that this has been tested, or to get the fix here.

CC: @QuLogic